pkg/api/registered/registered.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package to keep track of API Versions that should be registered in api.Scheme.
+// This is adapted from upstream for use in client/server version negotiation.
+package registered
+
+import (
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+// List of registered API versions.
+// The list is in the order of most preferred to the least.
+var RegisteredVersions []string
+
+func init() {
```

Comment:
This is just to set up `RegisteredVersions` for use by codecs?  If so, I don't see a reason for this `init`